recipes/osqp/all/conanfile.py

```diff
@@ -18,12 +18,29 @@ class OsqpConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "printing": [True, False],
+        "profiling": [True, False],
+        "ctrlc": [True, False],
+        "dfloat": [True, False],
+        "dlong": [True, False],
+        "coverage": [True, False],
+        "mkl_paradisio": [True, False]
```

Comment:
I would suggest skipping the `mkl_paradisio` option as it is being replaced by `OSQP_ALGEBRA_BACKEND` in the upcoming v1 release and would have to be deprecated. https://github.com/osqp/osqp/blob/master/CMakeLists.txt#L77-L79